internal/runbits/requirements/requirements.go

```diff
@@ -415,16 +415,17 @@ func (r *RequirementOperation) updateCommitID(commitID strfmt.UUID) error {
 		return locale.WrapError(err, "err_package_update_commit_id")
 	}
 
-	bp := model.NewBuildPlannerModel(r.Auth)
-	expr, err := bp.GetBuildExpression(r.Project.Owner(), r.Project.Name(), commitID.String())
-	if err != nil {
-		return errs.Wrap(err, "Could not get remote build expr")
-	}
+	if r.Config.GetBool(constants.OptinBuildscriptsConfig) {
+		bp := model.NewBuildPlannerModel(r.Auth)
+		expr, err := bp.GetBuildExpression(r.Project.Owner(), r.Project.Name(), commitID.String())
+		if err != nil {
+			return errs.Wrap(err, "Could not get remote build expr")
+		}
 
-	// Note: a commit ID file needs to exist at this point.
```

Comment:
I would like to keep this comment because it's important. This code block has shifted around the file over the last year.

Reply:
It's no longer relevant; the commit ID file no longer exists. We went back on that whole thing.